tools/jil/package.json

```diff
@@ -23,6 +23,7 @@
     "minimatch": "^3.0.3",
     "multiparty": "^4.1.2",
     "newrelic": "^4.11.0",
+    "node-fetch": "^3.2.10",
```

Comment:
FYI node v17.5+ introduced fetch (dev-dependencies) natively

Reply:
Nice!  that's a good question.  We'd have to bump a few things, like our GH actions as well, which i think run in v14.x